### ISSUE TITLE
Merkle patricia

### ISSUE THREAD

**Issue by laohanlinux** [exonum/exonum-doc#156]
Hi, not found about merkle tree detail doc, can add detail doc about it ?

**Comment by slowli** [comment]
Hi! Do you mean Merkle trees (used for lists) or Merkle Patricia trees (used for maps)? You can read about distinctions [here](https://exonum.com/doc/architecture/storage/#merklized-indices).

There is [a separate documentation page](https://exonum.com/doc/advanced/merkelized-list/) about usage of Merkle trees in Exonum. As for Merkle Patricia trees, at the time, there is no separate page in the docs, although there is an incomplete PR on this topic, #80. If it may help you, there *is* [library documentation on the subject](https://docs.rs/exonum/0.4.0/exonum/storage/proof_map_index/struct.ProofMapIndex.html).

**Comment by laohanlinux** [comment]
yes, i want to find merkle Patricia trees(used for maps), #80 is good
